### ISSUE TITLE
Convert part of step file surface to stl mesh

### ISSUE THREAD

**Issue by JulianWi21** [tpaviot/pythonocc-core#1013]
Hello everyone, 

is it possible to convert only a part of a step file, like a specific surface, to a mesh and a stl file?
In my step file I have got various faces with a certain colour that reflect my regions of interest.

Thanks in advance.

**Comment by tpaviot** [comment]
Yes it is. Traverse all subshapes and faces, then extract the relevant face to an stl file.
